src/class_objectProcessorQueue.py

```diff
@@ -1,23 +1,32 @@
+"""
+src/class_objectProcessorQueue.py
+=================================
+"""
 import Queue
 import threading
 import time
 
+
 class ObjectProcessorQueue(Queue.Queue):
+    """A queue for processsing message objects"""
     maxSize = 32000000
 
     def __init__(self):
         Queue.Queue.__init__(self)
         self.sizeLock = threading.Lock()
-        self.curSize = 0 # in Bytes. We maintain this to prevent nodes from flooing us with objects which take up too much memory. If this gets too big we'll sleep before asking for further objects.
+        # in Bytes. We maintain this to prevent nodes from flooing us with objects
+        # which take up too much memory. If this gets too big we'll sleep before
+        # asking for further objects.
```

Comment:
use `#:`

Reply:
https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.queues.html#pybitmessage.queues.objectProcessorQueue

Reply:
Can you please elaborate more about this

Reply:
It seems I already pointed you to the relevant chapter of the sphinx doc: http://www.sphinx-doc.org/en/1.8/usage/extensions/autodoc.html#directive-autodata